integration-cli/docker_cli_inspect_test.go

```diff
@@ -9,36 +9,38 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/integration/checker"
```

Comment:
So we should use `checker` if possible?  and will deprecate the `go-check` in the future?